lighthouse-core/lib/script-helpers.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
```

Comment:
2020 sucked lets go to 2022
```suggestion
 * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
```